### PR TITLE
Replace DateTime by DateTimeInterface

### PR DESCRIPTION
**Pull request by bobbybouwmann**
See #12

### CODE REVIEW

File: tests/OpeningHoursTest.php [hunk 1]
```diff
@@ -98,6 +98,10 @@ public function it_can_determine_that_its_open_at_a_certain_date_and_time()
         $this->assertTrue($openingHours->isOpenAt($shouldBeOpen));
         $this->assertFalse($openingHours->isClosedAt($shouldBeOpen));
 
+        $shouldBeOpenAlternaiveDate = date_create_immutable('2016-09-26 11:12:13.123456');
```
Comment (bobbybouwmann) [inline tests/OpeningHoursTest.php]: Oeps

**Comment by bobbybouwmann** [comment]
Can you add the hacktoberfest label? :D

**Comment by bobbybouwmann** [comment]
Damn you are quick!

**Comment by freekmurze** [comment]
Thank you, hope you'll get a t-shirt at the end of the month

**Comment by bobbybouwmann** [comment]
This was my 4th PR, but I keep doing PR's ;)

**Pull request merged**